src/graphql/Bug.graphql

```diff
@@ -269,6 +269,15 @@ input BugSearch {
   # Search the "Status Whiteboard" field on bugs for a substring. Works the same as the `summaries`
   # field, but searches the Status Whiteboard field.
   whiteboards: [String]
+
+  # The name of the field being searched as advanced query parameters
+  field: String,
```

Comment:
Doing it this way won't allow us to use f2 o2 v2, f3 o3 v3, etc... right?

We can make that an array instead of string but might need some extra logic to extract and group the fields, operators, and value. I would honestly just rename them to `field1`, `operator1`, and `value1` unless @eliperelman has a better idea.

Reply:
what about this?  `fields: [String], operators: [String], values: [String]`

Reply:
Sure.

Reply:
I like that as well.